docker_settings.yaml

```diff
@@ -1,10 +1,121 @@
-# example file to run mimir in docker, the file path must be changed
-osm_file: /data/osm/luxembourg-latest.osm.pbf
+# example file to run mimir in docker
+
```

Comment:
side note but can you add 
```
geocoder_tester_region: luxembourg
```

to the file ? this way it's a complete luxembourg example